Installation docs update for Debian 8 and RHEL 7

anmenaga opened the pull request:
This adds separate installation instructions sections for Debian 8 and RHEL 7.

Related to #1903.

Commit:
Debian8 update

Commit:
RHEL7 update

Commit:
Main page update

Commit:
Fixed 'MD024 Multiple headers with the same content'

SteveL-MSFT commented on docs/installation/linux.md:
```diff
@@ -127,11 +129,59 @@ sudo apt-get remove powershell
+
+### Installation via Package Repository - Debian 8
+
+PowerShell Core, for Linux, is published to package repositories for easy installation (and updates).
```
It might be worth adding that the Ubuntu14 deb is compatible and tested on Deb8 so that people aren't confused by ubuntu14 being in the URL

SteveL-MSFT replied:
@joeyaiello since Andrew is out right now, I made the change in this branch.  can you review the text below and approve for provide feedback?

SteveL-MSFT reviewed (changes requested):
See comment about Ubuntu14 compatibility

SteveL-MSFT committed:
Update linux.md

Added explicit text about compatibility of Ubuntu14.04 pkg on Debian8 to avoid confusion about URLs

SteveL-MSFT committed:
Update linux.md

joeyaiello reviewed (approved):
LGTM
